### PR TITLE
New branding

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/components/atoms/Title.jsx [hunk 1]
```diff
@@ -37,39 +37,22 @@ const Headline = ({
...
-        case 5: {
-            return <h5 {...attributes}>{children}</h5>;
-        }
-
-        case 6: {
-            return <h6 {...attributes}>{children}</h6>;
-        }
         default: {
             return <h1 {...attributes}>{children}</h1>;
         }
```
Comment (sgibson47) [inline src/components/atoms/Title.jsx:49]: Should there be cases for `'xl'` and  `'xxl'`?

File: src/styles/_colors.scss [hunk 2]
```diff
@@ -55,74 +30,87 @@ $overlays: (
...
-        @each $name, $attr in $gradients {
-            &-#{$name}--horz {
-                @include linear-gradient(to right, $attr);
-            }
-            &-#{$name}--vert {
-                @include linear-gradient(to bottom, $attr);
-            }
-        }
+        bg-brand-primary $color-brand-primary,
+    bg-brand-secondary $color-brand-secondary,
```
Comment (sgibson47) [inline src/styles/_colors.scss:36]: Do these not need colons, : ?

File: src/styles/_utilities.scss [hunk 3]
```diff
@@ -93,11 +93,10 @@ $customNamedClasses: (
         bg-sz-ct 'contain'
     ),
     border-radius: (
-        br-xs $border-radius-XS,
-        br-sm $border-radius-SM,
-        br-md $border-radius-MD,
-        br-lg $border-radius-LG,
-        br-xl $border-radius-XL,
+        br-sm $border-radius-sm,
+        br-md $border-radius-md,
```
Comment (sgibson47) [inline src/styles/_utilities.scss:97]: same question as above, maybe I'm just not familiar with this syntax

File: src/components/atoms/Title.jsx (hunk 1 above)
Comment (mgorman87) [reply src/components/atoms/Title.jsx:49]: Those sizes should be used for page titles so in theory they'll always be H1's.

File: src/styles/_colors.scss (hunk 2 above)
Comment (mgorman87) [reply src/styles/_colors.scss:36]: No need, there's a function on 112 that loops through these

File: src/styles/_utilities.scss (hunk 3 above)
Comment (mgorman87) [reply src/styles/_utilities.scss:97]: Same as the background classes

**Pull request merged**